Jellyfin.Plugin.TelegramNotifier/Configuration/Web/configPage.js

```diff
@@ -127,17 +127,27 @@ export default function (view) {
                     document.querySelector('#ServerUrl').value = config.ServerUrl;
                     document.querySelector('#BotToken').value = userConfig.BotToken;
                     document.querySelector('#ChatId').value = userConfig.ChatId;
+                    document.querySelector('#MessageTest').value = userConfig.MessageTest;
                     document.querySelector('#EnableUser').checked = userConfig.EnableUser;
                     document.querySelector('#SilentNotification').checked = userConfig.SilentNotification;
                     document.querySelector('#DoNotMentionOwnActivities').checked = userConfig.DoNotMentionOwnActivities;
+                    document.querySelector('#MessageItemAddedPrefix').value = userConfig.MessageItemAddedPrefix;
+                    document.querySelector('#MessageItemAddedSuffix').value = userConfig.MessageItemAddedSuffix;
+                    document.querySelector('#EnableItemAddedImage').checked = userConfig.EnableItemAddedImage;
+                    document.querySelector('#EnableItemAddedOverview').checked = userConfig.EnableItemAddedOverview;
                     TelegramNotifierConfig.notificationType.loadNotificationTypes(userConfig);
                 } else {
                     document.querySelector('#ServerUrl').value = config.ServerUrl;
                     document.querySelector('#BotToken').value = '';
                     document.querySelector('#ChatId').value = '';
+                    document.querySelector('#MessageTest').value = "[Jellyfin] Test message: \n 🎉 Your configuration is correct ! 🥳";
                     document.querySelector('#EnableUser').checked = false;
                     document.querySelector('#SilentNotification').checked = false;
                     document.querySelector('#DoNotMentionOwnActivities').checked = false;
+                    document.querySelector('#MessageItemAddedPrefix').value = "🎬";
```

Comment:
Let the user decide. Default with empty string because notifiers already have custom unicodes. But interesting feature with for example "[Server Name]".

```
document.querySelector('#MessageItemAddedPrefix').value = "";
document.querySelector('#MessageItemAddedSuffix').value = "";
```

Reply:
I agree with that.